caikit/runtime/service_generation/protoable.py

```diff
@@ -91,6 +99,30 @@ def handle_protoables_in_union(arg_type: Type) -> Type:
     log.debug("Skipping non-protoable argument type [%s]", arg_type)
 
 
+def get_union_list_type(field_name: str, union_protoables: List) -> Type[DataBase]:
+    """Create a union from list type objects"""
+    common_dm_package = caikit.interfaces.common.data_model
+    param_list = []
+    for arg in union_protoables:
+        if get_origin(arg) == list:
+            arg_type = get_args(arg)[0]
+            arg_name = f"{arg_type.__name__.capitalize()}Sequence"
+            if not hasattr(common_dm_package, arg_name):
+                raise AttributeError(
+                    f"Unable to find {arg_name} in {common_dm_package}"
+                )
+            do = getattr(common_dm_package, arg_name)
+            param_list.append(
+                Annotated[
+                    do,
+                    OneofField(field_name + "_" + arg_type.__name__ + "_" + "sequence"),
+                ]
+            )
+        else:
+            param_list.append(arg)
+    return Union[tuple(param_list)]
```

Comment:
Is this portable to all of our supported python versions? I had the same thing [over in http server](https://github.com/caikit/caikit/blob/main/caikit/runtime/http_server.py#L466) and my vscode built-in linting went crazy about it